include/zephyr/bluetooth/hci_types.h

```diff
@@ -344,16 +344,16 @@ struct bt_hci_cmd_hdr {
 /* Obtain OCF from OpCode */
 #define BT_OCF(opcode)                          ((opcode) & BIT_MASK(10))
 
-#define BT_HCI_OP_INQUIRY                       BT_OP(BT_OGF_LINK_CTRL, 0x0001)
+#define BT_HCI_OP_INQUIRY                       BT_OP(BT_OGF_LINK_CTRL, 0x0001) /* 0x0401 */
```

Comment:
Not sure if wanted, but if you modify them to be like
```suggestion
#define BT_HCI_OP_INQUIRY                       BT_OP(BT_OGF_LINK_CTRL, 0x0001) /**< 0x0401 */
```

Then they will also show up in the documentation.